.github/workflows/release.yaml

```diff
@@ -56,10 +53,25 @@ jobs:
         uses: actions/download-artifact@v3
 
       - name: Install wheel
-        run: pip install artifact/*.whl
+        run: python -m pip install artifact/*.whl
       
-      - name: Test application runs correctly
-        run: python -m superduperdb serve
+        # smoke-tests that check imports
+      - name: Check imports
+        # these modules cover most of the imports for the core package
+        # Note: if these modules are renamed, please update this list
+        run: |
+          python -m superduperdb.db.base.db
+          python -m superduperdb.db.base.backends
+          python -m superduperdb.container.model 
+
+      - name: Set version for tagging
+        id: set-version
+        run: |
+          export PACKAGE_VERSION=$(python -c "import superduperdb as s; print(s.__version__)")
+          echo "::set-output name=package_version::${PACKAGE_VERSION}"
```

Comment:
`::set-output` is out of date, we should use `GITHUB_OUTPUT`  in our CI https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Thank you! This is fixed now.